assets/app/view/game/spreadsheet.rb

```diff
@@ -118,22 +118,22 @@ def render_player_or_history
       end
 
       def render_history(corporation)
-        or_history(@game.all_corporations).map do |x|
-          render_or_history_row(corporation.operating_history, corporation, x)
+        or_history(@game.all_corporations).map do |turn_round|
+          render_or_history_row(corporation.operating_history, corporation, turn_round)
         end
       end
 
-      def render_or_history_row(hist, corporation, x)
-        if hist[x]
+      def render_or_history_row(hist, corporation, turn_round)
+        if hist[turn_round]
           revenue_text, alpha =
-            case (hist[x].dividend.is_a?(Engine::Action::Dividend) ? hist[x].dividend.kind : 'withhold')
+            case (hist[turn_round].dividend.is_a?(Engine::Action::Dividend) ? hist[turn_round].dividend.kind : 'withhold')
```

Comment:
This line is too long now, and an alternative is needed. what would that look like?

Reply:
How about assigning `div = hist[turn_round].dividend` on the previous line and then using that? It would make the code look nicer in any case.

Reply:
That would fail. But there are other ways.
https://18xxgames.slack.com/archives/C012K0CNY5C/p1617108056426900?thread_ts=1617103417.422800&cid=C012K0CNY5C